support/src/main/java/org/jenkinsci/plugins/workflow/support/storage/SimpleXStreamFlowNodeStorage.java

```diff
@@ -150,6 +151,7 @@ private Tag(@Nonnull FlowNode node, @Nonnull List<Action> actions) {
     private static final Field FlowNode$parentIds;
 
     static {
+        XSTREAM.addCompatibilityAlias("com.cloudbees.workflow.cps.checkpoint.NotExecutedNodeAction", NotExecutedNodeAction.class);
```

Comment:
I feel childish finding only formatting things, anyway, but isn't it wrongly indented, or is it GH that's not rendering it correctly? (Didn't check locally)